python/config.py.in

```diff
@@ -2,13 +2,12 @@ import os
 import sys
 
 shlib_ext = "@LTDL_SHLIB_EXT@"
-llvm_obj_root = "@LLVM_BINARY_DIR@"
 
 if sys.platform.startswith("win32"):
-    mlir_runner_utils_dir = os.path.normpath(os.path.join(llvm_obj_root, "bin"))
+    mlir_runner_utils_dir = os.path.normpath(os.path.join("@LLVM_BINARY_DIR@", "bin"))
```

Comment:
why are we "inlining" this variable? I think using a `llvm_obj_root` to hold the dir name would be better?

Reply:
fixed